### PR TITLE
Added load error text if window.WebChat is undefined

### PR DESCRIPTION
**Pull request by ghost**
## Description
If for some reason an error occurs while loading the chatbot such as the `window.WebChat` object being undefined we would like to be to show an error message.

## Testing done
This was done manually by adding `delete window.WebChat;` before it was called. 

## Screenshots
image

## Acceptance criteria
N/A

### CODE REVIEW

File: src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export default function ChatbotLoadError(_props) {
```
Comment (katieho-va) [inline src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:4]: Do you need the parameter `_props` here?

File: src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  return (
+    <AlertBox
+      headline="Error Loading Chatbot"
+      content="We are sorry there was an issue loading the chatbot, please refresh the page to try again. If the problem persists there may be a caching issue, please clear your cache and refresh the page."
```
Comment (katieho-va) [inline src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:8]: Do we need or did we get content review on this message? Just curious

**Review by katieho-va — APPROVED** [review]

File: src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx (hunk 1 above)
Comment (shaunhubbard-va) [reply src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:4]: just habit, the short answer is no, I can remove

File: src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx (hunk 2 above)
Comment (shaunhubbard-va) [reply src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:8]: I dont know I am still new to the VA's way of doing things. @mchelen-gov mind weighing in here?

Comment (shefalinayak) [reply src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:8]: @DanielleThierryUSDSVA do you have input on this?

**Comment by karlbrown-va** [comment]
can you confirm that if, for example, the CDN is blocked, that this error will come up?

File: src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx (hunk 2 above)
Comment (karlbrown-va) [reply src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:8]: i think 'clear your cache' is not something most people would know how to do. Maybe pointing to a page with such instructions would be helpful as it's different for various browsers (e.g. triggering a force refresh)

**Comment by katieho-va** [comment]
> can you confirm that if, for example, the CDN is blocked, that this error will come up?

@karlbrown-va @shaunhubbard-va Just ran these changes locally on the branch and without the CDN link in `vagov-content`(simulating CDN being blocked because chatbot won't load), I do get this new Error component

**Comment by DanielleThierryUSDSVA** [comment]
Hi. Yes, error messages need content reviews - thank you for checking! I'm looking at this now.

**Comment by DanielleThierryUSDSVA** [comment]
Error message content below. For awareness, we do have some standard messaging options in our error messaging dictionary/style guide. For this one, I took one of the standard messages and added info about the browser/cache clearing messaging. This may be a good resource for the team to have for future error messaging needs: https://design.va.gov/patterns/messaging-dictionary.

(Note: I noticed that the dictionary has "Internet" initial capped. It should be lowercase, so I'll ask someone to fix that).

## Error message copy

**We can’t load the chatbot**

Please make sure you’re connected to the internet, and refresh this page to try again.

If it still doesn’t work, you may need to clear your internet browser’s history (sometimes called “cached data”). You can find how to do this within your browser’s privacy and security settings.

File: src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx (hunk 2 above)
Comment (mchelen-gov) [reply src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:8]: see https://github.com/department-of-veterans-affairs/vets-website/pull/12561#issuecomment-625428860 for updated text

**Comment by katieho-va** [comment]
Thank you @DanielleThierryUSDSVA! How does this copy look?

Screen Shot 2020-05-07 at 2 54 15 PM

File: src/applications/coronavirus-chatbot/createCoronavirusChatbot.js [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import recordEvent from 'platform/monitoring/record-event';
 import { GA_PREFIX } from './utils';
 import * as Sentry from '@sentry/browser';
+import React from 'react';
```
Comment (mchelen-gov) [inline src/applications/coronavirus-chatbot/createCoronavirusChatbot.js:4]: does `React` have to be imported here? it seems like only `ReactDOM` is being used

**Review by mchelen-gov — APPROVED** [review]

File: src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx (hunk 2 above)
Comment (mchelen-gov) [reply src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx:8]: see https://github.com/department-of-veterans-affairs/vets-website/pull/12561#issuecomment-625428860 for updated text

**Review by mchelen-gov — CHANGES_REQUESTED** [review]
see https://github.com/department-of-veterans-affairs/vets-website/pull/12561#issuecomment-625428860 for updated text

File: src/applications/coronavirus-chatbot/createCoronavirusChatbot.js (hunk 3 above)
Comment (katieho-va) [reply src/applications/coronavirus-chatbot/createCoronavirusChatbot.js:4]: `React` is still needed for including the `ChatbotLoadError` JSX component here

**Comment by katieho-va** [comment]
@mchelen-gov copy has been updated

**Review by mchelen-gov — APPROVED** [review]

**Review by theodur — APPROVED** [review]

**Pull request merged**